install/kubernetes/helm/istio-remote/Chart.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: v1
 name: istio-remote
 version: 1.1.0
 appVersion: 1.1.0
+kubeVersion: ">=1.10.0"
```

Comment:
Yes, have same concern as @sdake, what is the case if kubeVersion smaller than 1.10.0?